armi/physics/fuelCycle/tests/test_fuelHandlers.py

```diff
@@ -497,20 +497,28 @@ def test_buReducingAssemblyRotation(self):
 
     def test_buildRingSchedule(self):
         fh = fuelHandlers.FuelHandler(self.o)
-        schedule, widths = fh.buildRingSchedule(17, 1, jumpRingFrom=14)
 
-        # test 1
-        self.assertEqual(
-            schedule, [13, 12, 11, 10, 9, 8, 7, 6, 5, 4, 3, 2, 1, 14, 15, 16, 17]
-        )
-        self.assertEqual(widths, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
+        # simple divergent
+        schedule, widths = fh.buildRingSchedule(1, 9)
+        self.assertEqual(schedule, [9, 8, 7, 6, 5, 4, 3, 2, 1])
 
-        # test 2
+        # simple with 1 jump
+        schedule, widths = fh.buildRingSchedule(9, 1, jumpRingFrom=6)
+        self.assertEqual(schedule, [5, 4, 3, 2, 1, 6, 7, 8, 9])
+        self.assertEqual(widths, [0, 0, 0, 0, 0, 0, 0, 0, 0])
+
+        # 1 jump plus auto-correction to core size
         schedule, widths = fh.buildRingSchedule(1, 17, jumpRingFrom=5)
-        self.assertEqual(
-            schedule, [17, 16, 15, 14, 13, 12, 11, 10, 6, 7, 8, 9, 5, 4, 3, 2, 1]
-        )
-        self.assertEqual(widths, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
+        self.assertEqual(schedule, [6, 7, 8, 9, 5, 4, 3, 2, 1])
+        self.assertEqual(widths, [0, 0, 0, 0, 0, 0, 0, 0, 0])
+
+        # crash on invalid jumpring
+        with self.assertRaises(ValueError):
+            schedule, widths = fh.buildRingSchedule(1, 17, jumpRingFrom=0)
+
+        # test 4: Mid way jumping
+        schedule, widths = fh.buildRingSchedule(1, 9, jumpRingTo=6, jumpRingFrom=3)
```

Comment:
Adding documentation in the fuel handler docs would be great to explain more of what is going on when before jumpRingTo and jumpRingFrom are used. I am able to follow the cases with jumpRingFrom but I don't quite understand why is this case (from right to left) we go 1, 2, 3 and then 6, 5, 4 and then 7, 8, 9. I get a bit confused as to why there is a reordering of 4 and 5 as well.